iceoryx_utils/test/moduletests/test_relative_pointer.cpp

```diff
@@ -126,7 +126,8 @@ TYPED_TEST(relativeptrtests, ConstrTests)
 
     {
         auto offset = ShmSize / 2;
-        void* adr = static_cast<uint8_t*>(memMap1.getMappedAddress()) + offset;
+        void* adr = reinterpret_cast<void*>(
```

Comment:
I think there are a few casts to much here ... the auto-merge might have get something wrong. the original line should be fine if memMap1 is replaced to memMap

Reply:
done